release-notes/5.0/5.0.0-preview1.md

```diff
@@ -0,0 +1,131 @@
+# Release Notes
+
+## Microsoft.Data.SqlClient 5.0.0-preview1.xxx released 9 March 2022
+
+This update brings the below changes over the previous release:
+
+### Added
+
+- Added SqlDataSourceEnumerator. [#1430](https://github.com/dotnet/SqlClient/pull/1430), [Read more](#sql-data-source-enumerator)
+- Added new attestation protocol `None` option to forgo enclave attestation when using VBS enclaves. [#1425](https://github.com/dotnet/SqlClient/pull/1425) and [#1419](https://github.com/dotnet/SqlClient/pull/1419), [Read more](#new-attestation-protocol-none)
+- Added a new AppContext switch to suppress insecure TLS warnings. [#1457](https://github.com/dotnet/SqlClient/pull/1457), [Read more](#suppress-insecure-tls-warning)
```

Comment:
```suggestion
- Added a new AppContext switch to suppress insecure TLS warnings. [#1457](https://github.com/dotnet/SqlClient/pull/1457), [Read more](#suppress-insecure-tls-warnings)
```

Reply:
there is only one type of  warning regarding that matter, plus this will just work as an anchor to read more section it wont show on the screen.

Reply:
link is broken, title says 'warnings'.

Reply:
fixed.